pkg/sql/opt/xform/join_funcs.go

```diff
@@ -480,6 +480,18 @@ func (c *CustomFuncs) generateLookupJoinsImpl(
 		lookupJoin.Cols = lookupJoin.LookupExpr.OuterCols().Difference(projectionCols)
 		lookupJoin.Cols.UnionWith(inputProps.OutputCols)
 
+		//var requiredCols opt.ColSet
+		//requiredCols.UnionWith(t.Relational().OutputCols)
+		//requiredCols.UnionWith(t.AllLookupFilters.OuterCols())
+		//requiredCols.UnionWith(t.On.OuterCols())
+		//requiredCols.UnionWith(t.KeyCols.ToSet())
+		//requiredCols.UnionWith(t.LookupExpr.OuterCols())
+		//requiredCols.UnionWith(t.RemoteLookupExpr.OuterCols())
+		//idx := m.Metadata().Table(t.Table).Index(t.Index)
+		//for i := range t.KeyCols {
+		//	requiredCols.Add(t.Table.ColumnID(idx.Column(i).Ordinal()))
+		//}
```

Comment:
Looks like this was mistakenly included.

Reply:
My mistake, removed it.